pyproject.toml

```diff
@@ -89,6 +83,14 @@ norecursedirs = [
 
 # ==== djLint ====
 
+[dependency-groups]
```

Comment:
I would expect this to be sorted higher in the file, I guess support hasn't landed yet in `pyproject-fmt`

Reply:
Gave this a try https://github.com/tox-dev/toml-fmt/pull/16